docker/debian9/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM debian:9 as builder
```

Comment:
Note that multi-stage builds require a recent version of docker. Make sure where you plan to build supports this.

Reply:
Thank you for your comment. At the moment we build docker images locally but contacted with @hortonworks/releng team about upgrading docker on jenkins slaves.